site/source/user-manual/service-guides/lightning.rst

```diff
@@ -8,10 +8,163 @@ Lightning
+
+Running a Node and Making a Channel with Start9
+_______________________________________________
+
+#. First, ensure that you have Bitcoin Core installed, running, and synced:
+
+   .. figure:: /_static/images/lightning/bitcoin-synced.png
+    :width: 60%
+
+#. Install a lightning node. There are two options we offer on the `Start9 marketplace <marketplace.start9.com>`_ - LND and Core-Lightning. In this guide we're going to use Core-Lightning (CLN). Though you can use LND and the process will be almost exactly the same.
+
+   .. figure:: /_static/images/lightning/two-impls-marketplace.png
+    :width: 60%
+
+#. Install one of the above lightning implementations - as mentioned in this guide we'll use CLN.
+
+   .. figure:: /_static/images/lightning/cln-installing.png
+    :width: 60%
+
+#. You'll see CLN say **"Needs Config"**. Click "**Configure**":
+
+   .. figure:: /_static/images/lightning/cln-needs-config.png
+    :width: 60%
+
+#. You can leave the settings as their default values and hit **"save"**.
+
+   .. figure:: /_static/images/lightning/cln-save-config.png
+    :width: 60%
+
+#. Now hit **"Start"** and wait for CLN to sync up to the network. This may take a few hours.
+
+   .. figure:: /_static/images/lightning/cln-syncing.png
+    :width:
+
+#. Once the **"Synced"** health check turns green (as below) you can proceed to the next step.
+
+   .. figure:: /_static/images/lightning/cln-green.png
+    :width: 60%
+
+#. To interact with your node we will use Ride The Lightning (RTL) - this is a service that provides a graphical user interface for our lightning node.
+
+    This will work with either (or both!) lightning implementations.
+
+      .. figure:: /_static/images/lightning/rtl-in-marketplace1.png
+        :width: 60%
+
+#. Install it and click on **"Configure"** just like with CLN.
+
+   .. figure:: /_static/images/lightning/rtl-needs-config.png
+    :width: 60%
+
+#. It will default to LND. In this case we are using a CLN node instead, so we will change the default setting as shown:
+
+   .. figure:: /_static/images/lightning/rtl-config-lnd.png
+    :width: 60%
+
+   Change to Core Lightning (CLN) and hit OK
+
+   .. figure:: /_static/images/lightning/rtl-change-to-cln.png
+    :width: 60%
+
+   Hit **"Save"**
+
+   .. figure:: /_static/images/lightning/rtl-config-save.png
+    :width: 60%
+
+#. Now hit **"Start"**
+
+   .. figure:: /_static/images/lightning/rtl-start.png
+    :width: 60%
+
+#. With RTL started, click **"Properties"**
+
+   .. figure:: /_static/images/lightning/rtl-click-properties.png
+    :width: 60%
+
+#. Copy the automatically generated password
+
+   .. figure:: /_static/images/lightning/rtl-copy-pass.png
+    :width: 60%
+
+#. Head back to the RTL service and click **"Launch UI"**
+
+   .. figure:: /_static/images/lightning/rtl-launch-ui.png
+    :width: 60%
+
+   Enter the copied password and log in:
```

Comment:
Make a note about adding your password to your password manager for convenience?